src/main/java/com/codingParkFun/rmsbackend/model/Student.java

```diff
@@ -1,50 +1,155 @@
 package com.codingParkFun.rmsbackend.model;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.util.Date;
 
 @Entity
 public class Student {
```

Comment:
Extend it ffrom user class

Reply:
sure. i forgot it.

Reply:
I tried but it didn't work. it gives null.